src/api/object_api.js

```diff
@@ -1373,6 +1374,8 @@ module.exports = {
                 is_latest: { type: 'boolean' },
                 delete_marker: { type: 'boolean' },
                 num_parts: { type: 'integer' },
+                start: { type: 'integer' },
+                end: { type: 'integer' },
```

Comment:
Should we name these more explicitly as `multipart_start`, `multipart_end`?
This might still be confusing a bit, unless we also rename `part_number` to `multipart_number`?

Reply:
Ok